File: appPython/AppThem.py
from SocketClient import SocketClient
from SocketServer import SocketServer
from Match import Match

class AppThem():


    def main():
        try:
            idMatch = 1;
            client = SocketClient("localhost", 12002)
            print("Cliente 'Them' concluido!\n")
            server = SocketServer(12003, idMatch)
            print("Servidor 'Them' concluido!\n")

            match = Match(1, 15, client, server)
            res: int = match.playMatch()
        except Exception as e:
            print("Error: " + str(e));

    if __name__ == "__main__":
        main()

File: appJava/src/main/java/game/AppThem.java
package game;

public class AppThem {
    public static void main(String[] args) throws Exception 
    {
        try
        {
            SocketClient client = new SocketClient("localhost", 12002);
            System.out.println("Cliente 'Them' concluido!");
            SocketServer server = new SocketServer(12003, 1);
            System.out.println("Servidor 'Them' concluido!");
            
            Match match = new Match(1, 5, client, server);
            int res = match.playMatch();
        }
        catch(Exception e)
        {
            System.out.println("Error: " + e.getMessage());
        }
    }
}
